Gemfile

```diff
@@ -6,6 +6,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 group :development, :test do
   gem 'byebug'
   gem 'factory_bot_rails'
+  gem 'pry-byebug'
```

Comment:
Can you extract this to a separate PR?  I really hate pry and I'd prefer if it was not part of the discussion on this issue.  Consider doing https://github.com/gerrywastaken/Gemfile.local maybe?

Reply:
d'oh - that was a mistake.  sorry!